### PR TITLE
Update dependencies, including webpack 4

### PR DESCRIPTION
**Pull request by bfred-it**
Webpack 4 released: https://medium.com/webpack/webpack-4-released-today-6cdb994702d4

Replaces #1113

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,13 +3,12 @@
 	"scripts": {
 		"lint": "xo && stylelint source/*.css",
 		"lint-fix": "xo --fix; stylelint --fix source/*.css",
-		"test": "npm run lint && cross-env BABEL_ENV=testing ava && run-s build:minified",
-		"build": "webpack",
-		"build:minified": "cross-env NODE_ENV=production webpack",
-		"watch": "webpack --watch",
+		"test": "npm run lint && cross-env BABEL_ENV=testing ava && run-s build",
+		"build": "webpack --mode production",
+		"watch": "webpack --mode development --watch",
```
Comment (sindresorhus) [inline package.json]: `--mode development` => `--mode=development`

Comment (bfred-it) [reply package.json]: What's the difference?

Comment (sindresorhus) [reply package.json]: Technically none, just more readable, as it's totally clear `development` is a value of `--mode` and not just an argument to `webpack`.

Comment (bfred-it) [reply package.json]: Ugly, but it makes sense 😄

**Pull request merged**